<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Головне вікно</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonAddSale">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Додати продаж</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddOrder">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Додати замовлення послуги</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShowSales">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Перегляд продаж</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShowProducts">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Перегляд проданих товарів</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShowServicesOrder">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Перегляд замовлень</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOut">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Вихід</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShowCustomers">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>141</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Перегляд клієнтів</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShowServices">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Перегляд наданих послуг</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
